eyes_selenium/applitools/selenium/eyes.py

```diff
@@ -383,13 +383,31 @@ def check(self, name, check_settings):
         :param check_settings: target which area of the window to check.
         :return: The match results.
         """
+        pass
+
+    @typing.overload
+    def check(self, check_settings):
+        # type: (SeleniumCheckSettings) -> None
+        """
+        Takes a snapshot and matches it with the expected output.
+
+        :param check_settings: target which area of the window to check.
+        :return: The match results.
+        """
+        pass
+
+    def check(self, check_settings_or_name=None, check_settings=None, name=None):
+        check_settings = merge_check_arguments(
```

Comment:
Actually, we could use some `**kwargs` here and get rid of multiple arguments. But currently, it's ok as well.